### DISCUSSION TITLE
The Hive: Assignee name

### DISCUSSION THREAD

**Discussion by arjoshi89** [Security-Onion-Solutions/securityonion#2197] [Unsupported Versions]
We have multiple people access to SO console with different login name.
All the issues which are escalated to the hive from SO console by different person appear as added by single person only under assignee?  
Is there any settings to change it?
![2020-12-03 13_12_09-TheHive - Cases](https://user-images.githubusercontent.com/17366610/101070384-4aefcd80-3569-11eb-8843-911d80412289.png)

**Comment by weslambert** [comment]
Thanks for reporting! We'll have to take a look at this.

**Reply by cyberkimh** [reply]
Yes we're noticing this issue also. I wonder if it's how we've configured it as I'm surprised more people haven't raised this as an issue?
